source/audio/Audio.cpp

```diff
@@ -110,6 +110,12 @@ namespace {
 	shared_ptr<Music> previousTrack;
 	int musicFade = 0;
 	vector<int16_t> fadeBuffer;
+
+	// The number of Pause vs Resume requests received.
+	int pauseChangeCount = 0;
+	// If we paused the audio multiple times, only resume it after the same number of Resume() calls.
+	// We start with -1, so when MenuPanel opens up the first time, it doesn't pause the loading sounds.
```

Comment:
Does this MenuPanel that opens the very first time also disappear? (If so, then the game will only pause audio if 2 or more MenuPanels are active after this change.)